src/App.js

```diff
@@ -234,7 +237,10 @@ function AppHeaderUser({
         <NavLink activeClassName="active" className="default-btn" to="/trade">Trade</NavLink>
       </div>
       <a href={accountUrl} target="_blank" rel="noopener noreferrer" className="App-cta small transparent App-header-user-account">
-        {shortenAddress(account, small ? 11 : 13)}
+        <div className="App-header-user-davatar">
+          <Davatar size={20} address={account} />
+        </div>
+        <div>{ensName || shortenAddress(account, small ? 11 : 13)}</div>
```

Comment:
Could you please check vertical alignment? Looks like with with ens name it's not centered

<img width="213" alt="Screenshot 2022-01-27 at 14 49 41" src="https://user-images.githubusercontent.com/95585424/151353391-044956f5-49e0-4966-b119-092de8b7fab5.png">

<img width="181" alt="Screenshot 2022-01-27 at 14 49 15" src="https://user-images.githubusercontent.com/95585424/151353395-ae578543-cd27-4341-bc67-3f9af1b7b0bb.png">

Reply:
![Screenshot 2022-01-27 at 5 41 35 PM](https://user-images.githubusercontent.com/69431456/151356433-343046df-86d7-4c9e-bb01-ecb07efa00a0.png)

@gdev8317 this is fixed now